receiver/aerospikereceiver/cluster/subsett_cluster.go

```diff
@@ -0,0 +1,58 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Should this file be `subset_cluster.go`?

Reply:
oops, changed "subsett" to "subset"